.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: Integration tests
```

Comment:
I think we need to move the `.skip-integration-steps` check here.

Reply:
removed